pulsar-common/src/test/java/org/apache/pulsar/common/compression/CompressorCodecTest.java

```diff
@@ -54,10 +72,10 @@
         };
     }
 
-    @Test(dataProvider = "codec")
-    void testCompressDecompress(CompressionType type, String compressedText) throws IOException {
+    @Test(dataProvider = "codecAndText")
+    void testCompressDecompress(CompressionType type, String sourceText) throws IOException {
```

Comment:
@gaoran10 Thanks for the fix. Is there any test that could reproduce the original failure without this fix?  If not, I would suggest we add a test to cover this.

Reply:
The compressed text param is not useful for this test, so I change the param from `compressedText` to `sourceText`, the purpose is to test complex text and no-repeated text, the complex text is used for the original test, the no duplicate text is used to verify this fix.